contracts/strategies/README.md

```diff
@@ -15,75 +15,77 @@ Provide a brief overview of the strategy and its purpose. Describe the high-leve
-7. Are there any limits on the number of recipients that can be registered? If so, what are those limits and how are they enforced?
-
-### Register Recipient Logic
-
-Describe the logic for registering recipients within the strategy. Explain the recipient status workflow, including the possible statuses (e.g., `None`, `Pending`, `Accepted`, `Rejected`) and the conditions under which the status changes. If additional functions are added to check or modify recipient status, mention them here.
-
-#### Questions to answer
-
-1. Are multiple registrations for a recipient allowed or just one?
-2. What happens in case of a second registration, either in the case of a single registration or multiple registrations?
-3. Can the address that receives the funds be different from the recipient's identity?
-4. Which address is returned by `registerRecipient` and represents the `receipientId`?
-5. Does the pool manager need to approve the registration, or is it automatically approved? If automatically, How? Describe in detail.
-6. What additional data is needed and passed during the registration process, apart from the recipient's address?
-7. Is the `registerRecipient` function accessible to anyone, or is it restricted to specific roles or addresses?
-8. Are there any specific conditions under which the `registerRecipient` function can or cannot be called?
-9. What steps are involved in changing the recipient's status from one state to another?
-10. Are there any additional functions available to check or modify the recipient's status?
-
-### Allocator Eligibility Logic
-
-Explain the logic for determining the eligibility of an allocator within the strategy. Specify any custom functions or conditions used to verify if an allocator is eligible to allocate.
+- **Recipient eligibility**
+    - What criteria determine the eligibility of a recipient to register in this strategy? Are there any specific requirements or conditions?
+    - Are there any additional custom functions or conditions for verifying recipient eligibility?
+- **Recipient information**
+    - How does a recipient identify themselves? Is it through an address like msg.sender, do they need an identity registered at the `Registry` contract, or another mechanism? 
+        - If through the Registry, can the pool creator set a custom registry?
+        - Which address is returned by `registerRecipient` and represents the `recipientId`?
+        - Can the address that receives the funds be different from the recipient's identity?
+    - What additional data is needed and passed during the registration process, apart from the recipient's address?
+- **Registering recipients**
+    - Is the `registerRecipient` function accessible to anyone, or is it restricted to specific roles or addresses?
+        - Are there any specific conditions under which the `registerRecipient` function can or cannot be called?
```

Comment:
I think this point doesn't belong to the bullet point above. the indentation level should be reduced, or?

Reply:
good catch — updated